src/GuiBase/Utils/KeyMappingManager.hpp

```diff
@@ -134,6 +134,7 @@ class RA_GUIBASE_API KeyMappingManager
     static Qt::MouseButtons getQtMouseButtonsValue( const std::string& buttonsString );
 
   private:
+    std::string defaultConfigFile;
```

Comment:
Please use `m_` prefix (gone due to some rebase?)

Reply:
sure, this must be done in the next accepted PR. Not necessarily this one :p

Reply:
well this data member is introduced here ! so the rename has to b done here (it's not present in current master KeyMappingManager.hpp)

Reply:
I think the point of @MathiasPaulin here was that this is not the final PR that will be merged but some kind of (advanced) draft, right?